bin/install-requirements.bash

```diff
@@ -6,4 +6,9 @@ export ANSIBLE_STDOUT_CALLBACK=yaml
 
 echo "This script will install all requirements"
 echo "Enter your password to start the install"
-ansible-playbook -e 'ansible_python_interpreter=/usr/bin/python3' --ask-become-pass --connection local --inventory 127.0.0.1, roles/get-requirements.yaml
+
+if [ "${#}" -ge 2 ] && [ "${2}" = "--user" ]; then
+  ansible-playbook -e 'ansible_python_interpreter=/usr/bin/python3' --ask-become-pass --extra-vars install_path="${HOME}/bin" roles/get-requirements.yaml
```

Comment:
Could you make this configurable? :pray: I put stuff in `.local/bin`.

Reply:
Does this work for you? https://github.com/elastisys/compliantkubernetes-apps/pull/2142/commits/703190a596f0573264c554404a26ae1eea6e4cc3

Reply:
Yeah :+1: